src/Sylius/Behat/Page/Admin/Taxon/UpdatePage.php

```diff
@@ -235,4 +237,22 @@ private function getImageElementByCode($code)
 
         return $inputCode->getParent()->getParent()->getParent();
     }
+
+    /**
+     * @param string $code
+     * @param string|null $path
+     */
+    private function attachImage($path, $code = null)
+    {
+        $filesPath = $this->getParameter('files_path');
+
+        $this->getDocument()->clickLink('Add');
+
+        $imageForm = $this->getLastImageElement();
+        if (null !== $code) {
```

Comment:
Instead of this if statement we add a code to `attachImageWithCode` function. It will reduce complexity.